### DISCUSSION TITLE
Uncaught SyntaxError: Unexpected token 'export'

### DISCUSSION THREAD

**Discussion by bellus93** [google/model-viewer#2901] [Q&A]
### Description

I have model viewer in an Angular application and i have this error in console 
Uncaught SyntaxError: Unexpected token 'export'

![Schermata 2021-10-21 alle 10 30 39](https://user-images.githubusercontent.com/19225097/138241137-1ef40fc6-8ed5-46d2-820a-00efe64800dc.png)

### Version

- model-viewer: v1.9.0

### Browser Affected

- [X] Chrome, version: 95.0.4638.54
- [X] Edge
- [X] Firefox
- [ ] IE
- [X] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [X] MacOS
- [X] Windows

**Comment by samaneh-kazemi** [comment]
Hi @bellus93 ,

Can you please provide a repro link ? Also, does this also happen with the previous versions of model-viewer? Maybe checkout similar threads with this error:

https://github.com/martinlaxenaire/curtainsjs/issues/14
